ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -728,6 +728,37 @@ abstract class AbstractHttpServiceIntegrationTest
       }: Future[Assertion]
   }
 
+  private[this] def randomTextN(n: Int) = {
+    import org.scalacheck.Gen
+    Gen
+      .buildableOfN[String, Char](n, Gen.alphaNumChar)
```

Comment:
Any reason to limit this to alphaNumChar?

Reply:
`\0` isn't allowed by the database, `filter` on Gens is unreliable, and this is the closest alternative at hand.